internal/services/kusto/kusto_eventhub_data_connection_resource.go

```diff
@@ -144,6 +144,12 @@ func resourceKustoEventHubDataConnection() *pluginsdk.Resource {
 				Default:      string(dataconnections.DatabaseRoutingSingle),
 				ValidateFunc: validation.StringInSlice(dataconnections.PossibleValuesForDatabaseRouting(), false),
 			},
+
+			"retrieval_start_date": {
+				Type:         pluginsdk.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringIsNotEmpty,
```

Comment:
should we use some validation here to ensure this is a valid timestamp?

```suggestion
				ValidateFunc:     validation.IsRFC3339Time,
```